src/meta/assessment/activityLogs/index.ts

```diff
@@ -21,8 +23,8 @@ const messageToKey: { [key in keyof typeof ActivityLogMessage]?: string } = {
   [ActivityLogMessage.originalDataPointCreate]: 'added',
   [ActivityLogMessage.originalDataPointRemove]: 'deleted',
   [ActivityLogMessage.originalDataPointUpdateDataSources]: 'updated',
-  [ActivityLogMessage.originalDataPointUpdateCommentExtentOfForest]: 'updated',
-  [ActivityLogMessage.originalDataPointUpdateCommentForestCharacteristics]: 'updated',
+  [ActivityLogMessage.originalDataPointUpdateCommentExtentOfForest]: 'edited',
+  [ActivityLogMessage.originalDataPointUpdateCommentForestCharacteristics]: 'edited',
```

Comment:
Should the used values be enums?
Why did we introduce new one over update?

Reply:
Let's use updated in this case. 

and let's not refactor to enums for now. we'll look into this later on